### PR TITLE
Consume Axe.Windows NuGet package

### PR DESCRIPTION
**Pull request by karanbirsingh**
#### Describe the change
This PR consumes the Axe.Windows NuGet package. This required the following steps:
- The existing Axe.Windows.Win32 namespace was changed to AccessibilityInsights.Win32 in order to avoid conflicts between the two namespaces
- Removing Axe.Windows projects and references so they can instead come from the NuGet package
- Installing the NuGet package to any projects that use Axe.Windows components
- Some cleanup

To make review a bit easier, here is [MSI/Product.wxs](https://github.com/microsoft/accessibility-insights-windows/pull/353/files#diff-1f8051a5d6658f41b012868c20b6ee74) and [here ](https://github.com/microsoft/accessibility-insights-windows/pull/353/files?file-filters%5B%5D=.config&file-filters%5B%5D=.cs&file-filters%5B%5D=.csproj&file-filters%5B%5D=.md&file-filters%5B%5D=.sln&file-filters%5B%5D=.wxs&hide-deleted-files=true&file-filters%5B%5D=.MD&file-filters%5B%5D=.a11ytest&file-filters%5B%5D=.bmp&file-filters%5B%5D=.hier&file-filters%5B%5D=.nuspec&file-filters%5B%5D=.resx&file-filters%5B%5D=.snapshot&file-filters%5B%5D=.tt)is a PR view that hides the deleted files.

#### PR checklist

- [x] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

**Comment by karanbirsingh** [comment]
@DaveTryon one outcome of this PR is that the check-automation script will need to be updated/checked in Axe.Windows rather than here. May be best for us to do that & update the validation steps immediately after this PR

**Comment by DaveTryon** [comment]
check-automation.ps1 is already in axe-windows. It should be removed from AIWin, preferably as part of this PR

---
In reply to: [491362226](https://github.com/microsoft/accessibility-insights-windows/pull/353#issuecomment-491362226) [](ancestors = 491362226)

**Comment by DaveTryon** [comment]
Eventually, I'd love to see some cleanup (mainly sorting) of products.wxs, but it can certainly be external to this PR

File: src/AccessibilityInsights.Extensions.GitHubAutoUpdate/Extensions.GitHubAutoUpdate.csproj [hunk 1]
```diff
@@ -40,6 +40,9 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (AhmedAbdoOrtiga) [inline src/AccessibilityInsights.Extensions.GitHubAutoUpdate/Extensions.GitHubAutoUpdate.csproj]: Not sure why this reference was added?

File: src/AccessibilityInsights.Extensions.Telemetry/Extensions.Telemetry.csproj [hunk 2]
```diff
@@ -43,6 +43,9 @@
     <Reference Include="Microsoft.ApplicationInsights, Version=2.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.ApplicationInsights.2.8.1\lib\net46\Microsoft.ApplicationInsights.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (AhmedAbdoOrtiga) [inline src/AccessibilityInsights.Extensions.Telemetry/Extensions.Telemetry.csproj]: Not sure why this reference was added?

File: src/AccessibilityInsights.SharedUx/SharedUx.csproj [hunk 3]
```diff
@@ -55,6 +79,9 @@
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Drawing" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (AhmedAbdoOrtiga) [inline src/AccessibilityInsights.SharedUx/SharedUx.csproj]: Not sure why this reference was added?

File: src/AccessibilityInsights.SharedUxTests/SharedUxTests.csproj [hunk 4]
```diff
@@ -82,6 +106,9 @@
     <Reference Include="System" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Drawing" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (AhmedAbdoOrtiga) [inline src/AccessibilityInsights.SharedUxTests/SharedUxTests.csproj]: Not sure why this reference was added?

File: src/AccessibilityInsights.WebApiHost/WebApiHost.csproj [hunk 5]
```diff
@@ -65,8 +89,17 @@
...
     </Reference>
+    <Reference Include="Sarif, Version=2.0.0.0, Culture=neutral, PublicKeyToken=21a5e83f6f5bb844, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Axe.Windows.0.1.0-prerelease\lib\net471\Sarif.dll</HintPath>
+    </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Collections.Immutable, Version=1.2.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Axe.Windows.0.1.0-prerelease\lib\net471\System.Collections.Immutable.dll</HintPath>
+    </Reference>
     <Reference Include="System.Core" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (AhmedAbdoOrtiga) [inline src/AccessibilityInsights.WebApiHost/WebApiHost.csproj]: Not sure why this reference was added?

File: src/AccessibilityInsights.WebApiHostTests/WebApiHostTests.csproj [hunk 6]
```diff
@@ -60,10 +84,22 @@
...
       <HintPath>..\packages\Owin.1.0\lib\net40\Owin.dll</HintPath>
     </Reference>
+    <Reference Include="Sarif, Version=2.0.0.0, Culture=neutral, PublicKeyToken=21a5e83f6f5bb844, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Axe.Windows.0.1.0-prerelease\lib\net471\Sarif.dll</HintPath>
+    </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Collections.Immutable, Version=1.2.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Axe.Windows.0.1.0-prerelease\lib\net471\System.Collections.Immutable.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (AhmedAbdoOrtiga) [inline src/AccessibilityInsights.WebApiHostTests/WebApiHostTests.csproj]: Not sure why this reference was added?

File: src/AccessibilityInsights.Extensions.GitHubAutoUpdate/Extensions.GitHubAutoUpdate.csproj (hunk 1 above)
Comment (DaveTryon) [reply src/AccessibilityInsights.Extensions.GitHubAutoUpdate/Extensions.GitHubAutoUpdate.csproj]: Code in this project consumes Newtonsoft, so this should have been here all along. Clean builds had build breaks about not having Newtonsoft, so I used VS to ensure that all Newtonsoft consumers were updated. VS inserted this reference as part of that process. It should stay

---
In reply to: [283034112](https://github.com/microsoft/accessibility-insights-windows/pull/353#discussion_r283034112) [](ancestors = 283034112)

File: src/AccessibilityInsights.Extensions.Telemetry/Extensions.Telemetry.csproj (hunk 2 above)
Comment (DaveTryon) [reply src/AccessibilityInsights.Extensions.Telemetry/Extensions.Telemetry.csproj]: Investigating

---
In reply to: [283034705](https://github.com/microsoft/accessibility-insights-windows/pull/353#discussion_r283034705) [](ancestors = 283034705)

**Comment by DaveTryon** [comment]
I sorted these

---
In reply to: [491391696](https://github.com/microsoft/accessibility-insights-windows/pull/353#issuecomment-491391696) [](ancestors = 491391696)

**Pull request merged**